CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [Unreleased]
+
+BUG FIXES:
+
+ - Ignore inaccessible files (such as emacs backup files) ([terraform-ls#1172](https://github.com/hashicorp/terraform-ls/issues/1067]))
+ - Fix crash when parsing JSON files (introduced in 0.30.0)  ([hcl-lang#202](https://github.com/hashicorp/hcl-lang/pull/202]))
```

Comment:
I think `0.30.0` would be better to replace with the appropriate extension version here.